### DISCUSSION TITLE
Label for axis

### DISCUSSION THREAD

**Discussion by shashank290521** [TanStack/react-charts#118] [General] [answered]
how can we show name of axis? means what x and y axis is representing how to give label for that? 
![pic](https://user-images.githubusercontent.com/65329059/92297505-87596080-ef5d-11ea-993d-611a43aa1bd8.png)

**Comment by tannerlinsley** [comment, marked as answer]
Due to a massive overhaul and rewrite of React Charts to TypeScript and a massively upgraded public API, this discussion is being marked as answered for triage. The latest version can be tested via the `react-charts@beta` tag. Documentation is in the process of being upgraded, but thankfully, everything is fully typed now, so you can explore the API that way for now.

If, *after testing or playing with the v3 beta*, you believe this discussion is still relevant or unanswered, feel free to mark it as such and continue the discussion.

**Reply by EarningsCall** [reply]
I don't think it's possible.

**Comment by narayanthapa15** [comment]
Any solution for this?
